front-end/cypress/e2e/F24/reports-f24-independent-expenditures.cy.ts

```diff
@@ -48,13 +48,13 @@ describe('Form 24 Independent Expenditures', () => {
 
     PageUtils.clickButton('Save');
     PageUtils.clickLink('Independent Expenditure');
-    cy.contains(individualContactFormData.first_name).should('exist');
-    cy.contains(individualContactFormData.last_name).should('exist');
+    cy.get('#first_name').should('have.value', individualContactFormData.first_name);
```

Comment:
Looks like all of these are also clicking on an IE in the list and checking the transaction detail page.  Could we add the `cy.contains('Address').should('exist');` to all the places you replaced the contains so we know we're on the details page

Reply:
Done